### DISCUSSION TITLE
I want to remove the alarm about http 4xx code in skywalking alarm, which is only effective for 5xx

### DISCUSSION THREAD

**Discussion by WB-jastme** [apache/skywalking#8293] [Q&A] [answered]
Spring cloud architecture, 4xx alarm called between services, I don't think it belongs to business alarm, how can i remove it?

**Comment by wu-sheng** [comment, marked as answer]
Which metrics do you mean?

**Reply by WB-jastme** [reply]
service_sla_rule

**Reply by WB-jastme** [reply]
Successful rate of service {name} is lower than 80% in 3 minutes of last 10 minutes
i got more 4xx http code,  i want disable 4xx ,only 5xx effective, can i?

**Reply by wu-sheng** [reply]
> service_sla = from(Service.*).percent(status == true);

According to the OAL script, it seems relating to the plugin codes. 

I agree that `4xx should not be considered as error in most cases`. You could find those plugins, and submit pull requests to change them, and remember to adjust test cases with them.

**Reply by WB-jastme** [reply]
ok,thanks!

**Reply by wangpf09** [reply]
i have the same question, but i dont know  `those plugins` means what?  
```
service_sla = from(Service.*).filter(httpResponseStatusCode != 404).percent(status == true);
```
i change the oal but it doesn't seem to take effect

**Reply by wu-sheng** [reply]
`filter(httpResponseStatusCode != 404)` filter means all other not 404 requests don't count.

This is not what you mean. You have to add a multiple condition function to replace `percent`
